File: src/main/java/org/jukeboxmc/event/player/PlayerCommandPreprocessEvent.java
package org.jukeboxmc.event.player;

import org.jukeboxmc.event.Cancellable;
import org.jukeboxmc.player.Player;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class PlayerCommandPreprocessEvent extends PlayerEvent implements Cancellable {

    private String command;

    /**
     * Creates a new {@link PlayerEvent}
     *
     * @param player who represents the player which comes with this event
     */
    public PlayerCommandPreprocessEvent( Player player, String command ) {
        super( player );
        this.command = command;
    }

    public String getCommand() {
        return this.command;
    }
}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/event/player/PlayerCommandPreprocessEvent.kt
package org.jukeboxmc.api.event.player

import org.jukeboxmc.api.event.Cancellable
import org.jukeboxmc.api.player.Player

class PlayerCommandPreprocessEvent(
        player: Player,
        private val command: String
) : PlayerEvent(player), Cancellable {

    fun getCommand(): String {
        return this.command
    }

}